distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestCatalog.java

```diff
@@ -2351,6 +2351,46 @@ public void testUpdateStorageCannotOverrideResourceUri() throws IOException {
         deleteMetacard(id);
     }
 
+    @Test
+    public void testCswMultiSortAsc() throws IOException {
+        final String sortCardId1 = ingestXmlFromResource("/sorttestcard1.xml");
+        final String sortCardId2 = ingestXmlFromResource("/sorttestcard2.xml");
+
+        String query = getFileContent("/csw-multi-sort-asc.xml");
+
+        ValidatableResponse validatableResponse = given().header(HttpHeaders.CONTENT_TYPE,
+                MediaType.APPLICATION_XML)
+                .body(query)
+                .post(CSW_PATH.getUrl())
+                .then();
+
+        validatableResponse.body(containsString(sortCardId1));
+        validatableResponse.body(not(containsString(sortCardId2)));
+
+        deleteMetacard(sortCardId1);
```

Comment:
Doesn't the catalog get cleared after every itest?  https://github.com/troymohl/ddf/blob/53290260a108ab415da55f36d57740ce2f2bb638/distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestCatalog.java#L197